renewal.conf.tpl

```diff
@@ -6,63 +6,78 @@ fullchain = :fullchain_path
 
 # Options and defaults used in the renewal process
 [renewalparams]
+no_self_upgrade = True
 apache_enmod = a2enmod
 no_verify_ssl = False
 ifaces = None
 apache_dismod = a2dismod
 register_unsafely_without_email = False
+apache_handle_modules = True
```

Comment:
Are these changes related to writing out `regr.json`?

Reply:
No. I pulled in the current renewal template from certbot.